conf/objects-list-all-on-one-page.js

```diff
@@ -0,0 +1,3 @@
+'use strict';
```

Comment:
I don't see this file required anywhere, so it cannot affect the files / certificates table. We want a fully working functionality for files / certificates after this PR is merged.